src/mono/wasm/runtime/library_mono.js

```diff
@@ -1290,7 +1319,7 @@ var MonoSupportLib = {
 				throw new Error (`"arguments" should be an array, but was ${request.arguments}`);
 
 			const objId = request.objectId;
-			let proxy;
+			let proxy; // Note: this variable is unused. Should it be removed?
```

Comment:
vote for removal

Reply:
Sorry, that was a mistake. Turns out it is used but just doesn't appear to be since it is used when the `fn_eval_str` variable is evaluated. I updated the comment to prevent future mix-ups like this